### PR TITLE
Add new option: Xie Tingfeng

### PR DESCRIPTION
**Pull request by moon2002**
well I added a new option to the question

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1966)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: Chinese/大话西游.md [hunk 1]
```diff
@@ -37,3 +37,5 @@
 [周星驰](Zhou/Zhou.md) ?
 
 [小熊维尼](xinwang/xin.md)?
+
+[谢霆锋](TingfengXie/TingfengXie.md)?
```
Comment (casey-collab) [inline Chinese/大话西游.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: Chinese/大话西游.md (hunk 1 above)
Comment (cbuckey-uda) [inline Chinese/大话西游.md]: If you just created the file `TingfengXie.md`, it looks like you may have forgotten to `git add` it.

**Comment by moon2002** [comment]
Problem fixed, please check, thanks

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**